dotcom-rendering/src/layouts/LiveLayout.tsx

```diff
@@ -353,9 +353,9 @@ export const LiveLayout = (props: WebProps | AppsProps) => {
 
 	const inUpdatedHeaderABTest =
 		article.config.abTests.updatedHeaderDesignVariant === 'variant';
-	// The test is being paused on liveblogs whilst we investigate an issue
 
-	const inTagLinkTest = false;
+	// The test is being paused on liveblogs whilst we investigate an issue
+	const shouldShowTagLink = false;
```

Comment:
There is a follow up ticket to implement this component for liveblogs, as designs have changed from the Euros